src/app/app.component.ts

```diff
@@ -140,20 +147,23 @@ export class AppComponent {
         this.filterModel = [];
         this.defaultFilter = [];
 
-        this.setDefaultFilters( appInjectDiv.getAttribute('data-defaultfilter') );
+        this.setDefaultFilters(appInjectDiv.getAttribute('data-defaultfilter'));
 
         // Setup search fields array
         this.searchFields = [];
-        this.searchFields.push( 'foo' );
-        this.searchFields.push( this.titleField );
-        this.searchFields.push( this.descriptionField );
-        this.searchFields.push( this.globalsService.DATE_FORMATTED );
+        this.searchFields.push(this.titleField);
+        this.searchFields.push(this.descriptionField);
+        this.searchFields.push(this.globalsService.DATE_FORMATTED);
+        const hiddenFields = this.filterHiddenSearch.split(',');
```

Comment:
@ESTEI9 Could you make this optional / add a check to see if `filterHiddenSearch` exists before parsing?